runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -667,7 +667,7 @@ NOTE: If you were to exit Neovim and start it again with `nvim TEST`, the file
 
  4. Type
 
-        `:w TEST`{vim}
+        `w TEST`{vim}
```

Comment:
I don't think this is correct

Reply:
Hello! These changes are explained in more detail here: https://github.com/neovim/neovim/pull/27689

Reply:
Specifically for this one: https://github.com/neovim/neovim/commit/0ae1ec868fc1d843514fd799a9ade92c0fac7e8a